core/trino-main/src/test/java/io/trino/operator/TestDynamicFilterSourceOperator.java

```diff
@@ -185,25 +192,25 @@ public void testCollectMultipleOperators()
         Operator op1 = createOperator(operatorFactory); // will finish before noMoreOperators()
         verifyPassthrough(op1,
                 ImmutableList.of(BIGINT),
-                new Page(createLongsBlock(1, 2)),
-                new Page(createLongsBlock(3, 5)));
+                new Page(createLongsBlock(10, 20)),
```

Comment:
Why are the values being multiplied by 10 in this test and in testCollectWithNulls ?

Reply:
To prevent them from being merged.
If you prefer, I can keep the original values - and change the expected results to check that discrete values' merging works.